codelabs/gRPC_Python_OpenTelemetry_Plugin/start_here/observability_greeter_server.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python implementation of the GRPC helloworld.Greeter server for observability codelab."""
+
+from concurrent import futures
+import logging
+import time
+
+import grpc
+import helloworld_pb2
+import helloworld_pb2_grpc
+
+_SERVER_PORT = "50051"
+
+
+class Greeter(helloworld_pb2_grpc.GreeterServicer):
+    def SayHello(self, request, context):
+        message = request.name
+        # CODELAB HINT: This sleep seems suspicious.
```

Comment:
sleeping is always a good thing 😋  Have you all tried it?

Reply:
Lol, of course :wink: